slack-api-client/src/test/java/test_with_remote_apis/methods/chat_Test.java

```diff
@@ -882,8 +882,8 @@ public void share_message_with_files_issue_647() throws Exception {
         assertThat(message.isOk(), is(true));
     }
 
-    // 2021-01-05 14:38:54,767 WARN [main] com.slack.api.methods.RequestFormBuilder The `text` argument is missing in the request payload for a chat.postMessage call - It's a best practice to always provide a text argument when posting a message. The `text` is used in places where `blocks` cannot be rendered such as: system push notifications, assistive technology such as screen readers, etc.
-    // 2021-01-05 14:38:55,055 WARN [main] com.slack.api.methods.RequestFormBuilder The `text` argument is missing in the request payload for a chat.update call - It's a best practice to always provide a text argument when posting a message. The `text` is used in places where `blocks` cannot be rendered such as: system push notifications, assistive technology such as screen readers, etc.
+    // 2021-01-05 14:38:54,767 WARN [main] com.slack.api.methods.RequestFormBuilder The top-level `text` argument is missing in the request payload for a chat.postMessage call - It's a best practice to always provide a `text` argument when posting a message. The `text` is used in places where `blocks` cannot be rendered such as: system push notifications, assistive technology such as screen readers, etc.
+    // 2021-01-05 14:38:55,055 WARN [main] com.slack.api.methods.RequestFormBuilder Additionally, the attachment-level `fallback` argument is missing in the request payload for a chat.postMessage call - To avoid this warning, it is recommended to always provide a top-level `text` argument when posting a message. Alternatively, you can provide an attachment-level `fallback` argument, though this is now considered a legacy field (see https://api.slack.com/reference/messaging/attachments#legacy_fields for more details).
```

Comment:
@filmaj Testing the logging output with the current code is not so easy that I added the evidence here (better than nothing!). Can you replace the logging output with your own one? You can find your logs at `slack-api-client/logs/console.log`.

Reply:
@seratch Done!